test/integration-tests/Uniswap.test.ts

```diff
@@ -72,7 +71,7 @@ describe('Uniswap V2 and V3 Tests:', () => {
 
       it('V2 exactIn, permiting the exact amount', async () => {
         const amountInDAI = expandTo18DecimalsBN(100)
-        const minAmountOutWETH = expandTo18DecimalsBN(0.03)
+        const minAmountOutWETH = expandTo18DecimalsBN(0.02)
```

Comment:
curious why the numbers in this test are changing? Is it just because we're forking from a more recent block?

Reply:
yeah just because we're forking from a different block and prices changed